spec/index.bs

```diff
@@ -3533,9 +3533,14 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
 <span id=texture-interface></span>
 </h3>
 
-Issue: Remove this definition: <dfn dfn>texture</dfn>
-
-One texture consists of one or more <dfn dfn>texture subresources</dfn>,
+A <dfn dfn>texture</dfn> is made up of {{GPUTextureDimension/1d}}, {{GPUTextureDimension/2d}},
+or {{GPUTextureDimension/3d}} arrays of data which can contain multiple values per-element to
+represent things like colors. Depending on the {{GPUTextureUsage}} they are created with, textures
+can be read or sampled from render pipelines, read and written to from compute pipelines, and set as
+attachments to render passes. Textures are stored in GPU memory with an layout optimized for
+multidimensional access rather than linear access.
```

Comment:
```suggestion
represent things like colors. Textures can be read and written in many ways, depending on the
{{GPUTextureUsage}} they are created with. For example, textures can be sampled, read, and written
from render and compute pipeline shaders, and they can be written by render pass outputs.
Internally, textures are often stored in GPU memory with a layout optimized for
multidimensional access rather than linear access.
```